GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -266,7 +266,7 @@ public FormBrowse(GitUICommands commands, BrowseArguments args)
             MainSplitContainer.Visible = false;
             MainSplitContainer.SplitterDistance = DpiUtil.Scale(260);
 
-            stashStagedToolStripMenuItem.Visible = GitVersion.Current.SupportStashStaged;
+            stashStagedToolStripMenuItem.Visible = Module.GitVersion.SupportStashStaged;
```

Comment:
Also need to be moved to InternalInitialize() as it may change when the module is updated.
A change still required to GitVersion, this was the first change that affected a supported setup...